src/requirements/fetcher.ts

```diff
@@ -138,8 +138,8 @@ const generateSemesterJSONs = async <T extends keyof Course>(
     );
     const courses: AllCourses<T> = {};
     courses[semester] = semesterCourses;
-    const fileRequirementsSrc = 'src/requirements/filtered-' + semester + '-courses.json';
-    const fileFunctionsSrc = 'functions/filtered-' + semester + '-courses.json';
+    const fileRequirementsSrc = `src/requirements/filtered-${semester}-courses.json`;
```

Comment:
This is fine, but either you or Theresa will probably have a fun merge conflict with this haha.